getting_started_tutorials/cuml_sklearn_colab_demo.ipynb

```diff
@@ -0,0 +1,4310 @@
+{
```

Comment:
<p>With the above changes, this step would become unnecessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>